_includes/datetime.html

```diff
@@ -9,6 +9,17 @@
   >
     {{ include.event.date | date:include.date_only_format }}
   </time>
+  {% if include.event.end_date %}
+    &mdash;
+    <time
+      {{ cancelled }}
+      class="time-span"
```

Comment:
Thoughts on making `time-span` prevent wrapping of the contained text? I Suspect that might look slightly better than the current wrapping result.

Reply:
![image](https://user-images.githubusercontent.com/6527489/163581509-46fa22e1-ad9e-41d7-8232-c5725b1ed1a9.png)

I agree this looks better, but I can't decide whether the trailing dash looks nice? Thoughts?

Reply:
I think that's preferable to the current wrapping.
IMO extra good would be that the end datetime is indented to align with the start (though that would look super weird if it was just the time that wrapped), so probably not worth it.